release/changes.md

```diff
@@ -1,2 +1 @@
-- [NEW] - Run tests against AGP 8.1.0-beta03
-- [NEW] - Run tests against AGP 8.2.0-alpha04
+- [NEW] TBD
```

Comment:
Why TBD?

Reply:
Fixed in 8c2f5dfdfa321fba470a7cc26ab6f21a62497a50